perma_web/perma/admin.py

```diff
@@ -572,7 +574,7 @@ class LinkUserAdmin(UserAdmin):
             'fields': ('email', 'password1', 'password2'),
         }),
     )
-    raw_id_fields = ['registrar', 'organizations']
```

Comment:
I wonder if we need the equivalent of this, for the `UserOrganizationAffiliationInline`. Have you run into `raw_id_fields` before? It is a performance optimization. Normally, when Django renders these on admin edit pages, it tries to give you a dropdown, from which you can select all the available options. So, in this case, it would have given you a dropdown with every single possible organization. For large tables, that turns out to be a performance problem... and can sometimes even stop the page from loading. Using this option tells Django just to print the related object's ID, instead, and gives you a number field you can change if you want.

It's probably worth checking with a production-like database, to make sure these pages performance are okay without a similar tweak to `UserOrganizationAffiliationInline`.

Reply:
Yep, I tested everything with a copy of prod db from yesterday and didn't run into any performance issues. I will read about raw_id_fields and see if I can incorporate it back just in case.